test/e2e/specs/specs-wpcom/wp-blocks__jetpack.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @group gutenberg
+ */
+import {
+	setupHooks,
+	BrowserHelper,
+	DataHelper,
+	GutenbergEditorPage,
+	TestAccount,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+const accountName = BrowserHelper.targetGutenbergEdge()
+	? 'gutenbergSimpleSiteEdgeUser'
+	: 'gutenbergSimpleSiteUser';
+
+describe( DataHelper.createSuiteTitle( 'Blocks: Jetpack Earn/Grow' ), function () {
```

Comment:
This spec is an identical migration from the Selenium counterpart, deleted below.

This spec **does not** insert blocks in the editor.